### DISCUSSION TITLE
Pages outside the main pages folder

### DISCUSSION THREAD

**Discussion by JSsantana** [vercel/next.js#41576] [Help]
I want to add pages in a different path and also keep nextJS looking into pages on the root directory. Example:

./pages/mypage.js

./folderA/folderB/pages/mypage.js

is that possible?

**Comment by thiswallz** [comment]
You could do that by creating a file within pages/ that reference a page in another place.

E.g. pages/other.js
```
export { default } from '../folderA/folderB/pages/mypage.js';
```
